src/log.js

```diff
@@ -202,7 +215,7 @@ export class Log {
   error(tag, var_args) {
     const error = this.error_.apply(this, arguments);
     if (error) {
-      this.win.setTimeout(() => {throw /** @type {!Error} */ (error);});
+      reportError(error);
```

Comment:
I'll merge for now. Considering to change our tests to always fail if an error was logged during execution (and requiring errors to be expected).